src/Utils/utils.ts

```diff
@@ -544,3 +544,29 @@ export const fahrenheitToCelsius = (fahrenheit: number) => {
 export const keysOf = <T extends object>(obj: T) => {
   return Object.keys(obj) as (keyof T)[];
 };
+
+//removes all symbols except _ and - (part of startCase)
+export const cleanString = (str: string): string => {
+  if (!str || str.length === 0) return "";
+  return str.replace(/[^a-zA-Z0-9_-]+/g, " ").trim();
+};
+
+// just capitalizes (as per startCase working) (part of startCase)
+export const capitalizeWords = (str: string): string => {
+  if (!str || str.length === 0) return "";
+  return str
+    .split(" ")
+    .map((word) => word.charAt(0).toUpperCase() + word.slice(1).toLowerCase())
+    .join(" ");
+};
+
+//equivalent to lodash omitBy
+export function omitBy<T extends Record<string, unknown>>(
+  obj: T,
+  predicate: (value: unknown) => boolean = (value) =>
+    value !== "" && value !== undefined && value !== null,
+): Partial<T> {
+  return Object.fromEntries(
+    Object.entries(obj).filter(([_, value]) => predicate(value)),
+  ) as Partial<T>;
+}
```

Comment:
pull out the `isEmpty` fn. out so that it could be re-used and would be cleaner.

```suggestion
export const isEmpty = (value: unknown) => {
  return value !== "" && value != null;
}

// equivalent to lodash omitBy
export function omitBy<T extends Record<string, unknown>>(
  obj: T,
  predicate: (value: unknown) => boolean = isEmpty,
): Partial<T> {
  return Object.fromEntries(
    Object.entries(obj).filter(([_, value]) => predicate(value)),
  ) as Partial<T>;
}
```